### ISSUE TITLE
Whole "editor" concept is rather confusing

### ISSUE THREAD

**Issue by engelhro** [mkorthof/chrupd#13]
Hello,

I discovered `chrupd` today, but immediately stumbled upon several inconsistencies. Maybe these could be fixed both within the script itself and also on the Github pages resp. the help.

The main problem seems to come from the inconsistent use of the term "editor". Sometimes it refers to the person which provides a chromium build (better term: "author"?) and sometimes to one of the various Chromium variants itself.

I guess that comes from the later added support for the ungoogled-chromium browser, but it now results in several contradicting outputs:

1. `chrupd -list` shows **two** rows with *editor* "Marmaduke", one with the *name* "Marmaduke" and another one with the *name* "Ungoogled".

2. What the script actually needs to work (and also shows in the help) is the *name*, not the *editor* as person:
    - either `Marmaduke` for the regular chromium browser from Marmaduke
    - or `Ungoogled` for the ungoogled-chromium browser from Marmaduke.

3. So wherever the readme or the script refer to an "editor" (which is also the parameter name!), it means *name* (=browser variant), not *editor* (as person).

4. Either way, the editor `Marmaduke-Ungoogled` as listed in the readme is completely wrong, as it mixes both *editor* (as person) and *name* (as browser variant) into one combined word, which itself isn't a valid value for the `editor` parameter…

Thanks a lot for clarifying and cleaning up these terminologies!

My suggestion would be to cleanly distinguish between `author` (person who provides a build) and `browser` (today's "name" = chromium variant provided) and get rid of that `editor` term completely, also as parameter name.

**Comment by mkorthof** [comment]
Hi,

Thanks for your suggestions.
Indeed this was done because one editor can have multiple Chromium releases.
Editor for releases can also change (e.g. Ungoogled).
The script uses https://chromium.woolyss.com, perhaps a visit makes it more clear.

Made 2 small changes to README and `-list`

**Comment by engelhro** [comment]
> Indeed this was done because one editor can have multiple Chromium releases.

I know, that's what I described and resulted in the mess I described.

> Editor for releases can also change (e.g. Ungoogled).

I know.

> The script uses https://chromium.woolyss.com, perhaps a visit makes it more clear.

I know, and the contents of that site and the facts are clear to me.

But how does that solve the issue I described with the inconsistent use of the "term" editor both in the script as well as in the documentation for the tool? This has nothing to do with the Woolyss site as such.

> Made 2 small changes to README and `-list`

Thanks! Still not sure whether this is enough in the long term, without accidentally introducing new confusion (or bugs) in the future in case of further changes to the list of authors and their builds.

And still the list presented in the README does not fit to the script (containing entries like "The Chromium Authors" or "ThumbApps" which do not appear in the help output, at least the one shown in the README). So the list of inconsistencies continues…
